test/cql-pytest/test_describe.py

```diff
@@ -460,6 +460,31 @@ def test_generic_desc(cql, random_seed):
             assert generic_tbl == desc_tbl
             assert generic_idx == desc_idx
 
+def test_generic_desc_user_defined(scylla_only, cql, test_keyspace):
```

Comment:
Please add a comment saying what this test is checking: namely it checks a Scylla extension where you are allowed to do "DESC ks.x" for types or functions instead of the longer "DESC TYPE ks.x" or "DESC FUNCTION ks.x". This test is marked scylla_only because it's a Scylla-only extension, not supported by Cassandra.

Reply:
added

Reply:
Ok, thanks, this comment is better than nothing, but I still think it will puzzle a developer who reads it a couple of years from now and hopes to understand what this test is about, and doesn't understand what phrases like "extended generic description" refers to. My suggestion above for a comment included a specific example to make it clearer than just saying a "generic" description.

By the way I forgot to ask this, but please also say in the comment that this test `reproduces #14170`. This information is also very helpful when trying to understand why a test exists or fails, or when we realize we forgot to backport something and don't know what.